### DISCUSSION TITLE
Testing the bug triage process

### DISCUSSION THREAD

**Discussion by chrisdavidmills** [mdn/mdn-community#208] [Content]
This issue is about testing the new bug triage process that has been published at https://developer.mozilla.org/en-US/docs/MDN/Contribute/Processes/Content_bug_triage.

In the [triagers list](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Processes/Content_bug_triage#decide_on_triagers) I have tentatively filled in some names of folk I'd like to put a little bit of time aside each week to help triage new documentation bugs — this includes me, @rachelandrew , @hamishwillee , @Elchi3 , @Rumyra , @Ryuno-Ki , and @caitmuenster and the WebExt team.

I'm anticipating that you should need about 30 mins per week at the most to triage bugs, depending on how busy your areas are.

the list of bugs you are looking to triage are marked by the labels "needs-triage" and "Content:_topic-area_", e.g. "Content:HTML". When you find your list, you need to go through each issue, remove the "needs-triage" label, and add other labels and information according to the process laid out in [Triage process for each issue](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Processes/Content_bug_triage#triage_process_for_each_issue).

Please help me by giving the process a bit of a test this week, and let me know if you have any feedback, concerns, or issues with the process.

Thanks.

**Comment by rachelandrew** [comment]
Could we have some kind of label for things we have decided are best solved with a script, or with a change to Yari etc.?

I'm looking at #3047 which is something we have discussed could be best solved with a script rather than people updating every page by hand, so would be good to track these. Likewise for issues raised here, which relate to something that actually needs fixing in the platform. When the iframes were all squashed would be a good example of that.

**Comment by chrisdavidmills** [comment]
@rachelandrew so you are potentially looking for something like "platform-issue" to draw attention to bugs that aren't going to be fixed in the content? I think better would be for y'all to just tag me when you find such a bug, and I'll move it to an appropriate place. `yari` for general platform bugs, and `mdn-minimalist` for platform styling bugs.

Sound good?

**Comment by hamishwillee** [comment]
@chrisdavidmills Should the process auto-assign to a triager? Just makes the filtering of "todo" a little easier.

**Comment by chrisdavidmills** [comment]
@hamishwillee so how would that work? When I go through and add say the "Content:DevTools" label, and the "needs-triage" label, it auto-adds you as the Assignee?

I worry that having such a process might cause would-be bug fixers to look at the newly-filed issue, see there's an assignee, and then not pursue fixing the bug as a result.

**Comment by hamishwillee** [comment]
@chrisdavidmills That's what I was thinking. You add the triage and remove your name.

Withdraw the suggestion; your concern is valid.

**Comment by Elchi3** [comment]
How do we avoid that people feel like they are "stealing" triage work from others when they really aren't? (at least for me, I'm more than happy if y'all steal ALL the work away from me).

See https://github.com/mdn/content/issues/2907#issuecomment-799914535 for an example of this situation.

**Comment by hamishwillee** [comment]
> How do we avoid that people feel like they are "stealing" triage work from others when they really aren't?

Perhaps explicitly add a note that the assigned person is only responsible for making sure that the notes are triaged in a timely manner; the work can be done by anyone.

**Comment by chrisdavidmills** [comment]
> > How do we avoid that people feel like they are "stealing" triage work from others when they really aren't?
> 
> Perhaps explicitly add a note that the assigned person is only responsible for making sure that the notes are triaged in a timely manner; the work can be done by anyone.

And also make sure that we don't give an issue more than one content area? That is the main reason this happened. I don't really see it as a huge problem anyway.

**Comment by Rumyra** [comment]
I timed myself this week - 40 mins for Web API triage 💪 (this doesn't include backlog only new)

I think a definition of tier docs would be good as I'm concluding a `p2` tag on a _lot_ of issues

**Comment by Ryuno-Ki** [comment]
Just noticed the mention here today.
Depending on my day-to-day job I spent sometimes one or two hours (per day) or skip other days entirely.
Still have ~1k unread GitHub notifications (with ~80 % of them being either content or yari).
It's fine. Don't worry about me.
I'm working my way backwards with focussing on closed issues and PRs first, then checking opened ones and issues with a mention on me.
My personal challenge is to keep track of those issues, where I still have a todo (be it only a „waiting for a response”). I believe, bugzilla has a needsinfo flag for that?
Right now, it seems to work to ping a person to bring it back up to get the issue / PR finalised.

**Comment by Ryuno-Ki** [comment]
I mean, we could also have a Markdown file in the repo with links like in https://github.com/11ty/eleventy/#major-roadmapped-features - which are essentially issue filters on GitHub. That could be part of the topic owners, for example.
